src/main/resources/swagger/rawls.yaml

```diff
@@ -4488,6 +4488,9 @@ definitions:
       stderr:
         type: string
         description: location of stderr
+      shardIndex:
+        type: integer
+        description: index of execution cluster instance
```

Comment:
I don't think this description is right. I think this shard index is the index of the call within a scatter block. Tracing the code, this number comes from the execution service and it can't know which instance it is.